src/cuda/graph_ops.cu

```diff
@@ -109,6 +111,64 @@ torch::Tensor GraphNormalize(torch::Tensor indptr,
   return out_data;
 }
 
+template <int BLOCK_SIZE>
```

Comment:
Move these kernels to `src/cuda/random_walk.cu`.

Reply:
OK